pyrasterframes/src/main/python/setup.py

```diff
@@ -72,6 +69,59 @@ def run(self):
                 print(traceback.format_exc())
 
 
+class PweaveDocs(distutils.cmd.Command):
+    """A custom command to run documentation scripts through pweave."""
+    description = 'Pweave PyRasterFrames documentation scripts'
+    user_options = [
+        # The format is (long option, short option, description).
+        ('files=', 'f', 'Specific files to pweave. Defaults to all in `docs` directory.'),
+    ]
+
+    def initialize_options(self):
+        """Set default values for options."""
+        # Each user option must be listed here with their default value.
+        self.files = filter(
+            lambda x: not x.name.startswith('_'),
+            glob(path.join(here, 'docs', '*.py'))
+        )
+
+    def finalize_options(self):
+        """Post-process options."""
+        import re
+        if isinstance(self.files, str):
+            self.files = filter(lambda s: len(s) > 0, re.split('\W+', self.files))
+
+    def run(self):
+        """Run pweave."""
+        import traceback
+        import pweave
+
+        for file in self.files:
+            name = path.splitext(path.basename(file))[0]
+            print(_divided('Running %s' % name))
+            try:
+                pweave.weave(
+                    file=str(file),
+                    doctype='markdown'
+                )
+            except Exception:
+                print(_divided('%s Failed:' % file))
+                print(traceback.format_exc())
+
+
+try:
+    with open(path.join(here, 'requirements.txt')) as f:
+        requirements = f.read().splitlines()
+except:
+    print("couldn't open requirements.txt")
+    requirements = [
+        'pytz',
+        'shapely',
+        'pyspark>=2.3',
+        'numpy>=1.7',
+        'pandas',
+    ]
+
 setup(
     name='pyrasterframes',
     description='RasterFrames for PySpark',
```

Comment:
We should think about this too. It's going to be the "tagline" in pypi.org page. Would like it to be very pithy explanation of project. pyspark can get away with saying they are the python API for spark, but we probably need to give more context to the Python user.  Some brainstorms

> Spatial raster support for Spark DataFrames

> Raster data in pyspark DataFrames

> Read, process, and analyze raster data in Spark DataFrames

Reply:
I think "geospatial" needs to be in there.

Reply:
Please take a look at the latest

Reply:
Not finding the latest tagline ideas 🤔 .

Ideally it would be fetched from this:

https://github.com/locationtech/rasterframes/blob/38e9ec6cbf58acb821e43eb87600ee70890e4a84/project/RFProjectPlugin.scala#L21

But not sure how to do that other than with environment variables, and that kinda sucks.

One idea is to add it to `RFBuildInfo` and fetch it with a fork to a JVM `main` that prints it. I just hate having the same info in multiple places.